tests/test_molecule.py

```diff
@@ -96,7 +96,7 @@ def test_gen_conformers():
         mol._generate_conformers()
 
     # Metal complexes must be completely bonded entities
-    with pytest.raises(ValueError):
+    with pytest.raises(Exception):
```

Comment:
Can you not test that it raises a specific error here?

Reply:
I'm not sure what it should raise, so was being lazy. Is there any advantage of testing for specific exceptions?

Reply:
Easier to see if something is going wrong. If there is another issue here that throws a different error, the test won't pick it up. Depends how likely you think the function is to be breaky

Reply:
Good point – will endeavour to test the right thing in future 👍

Reply:
Its more an issue if you're doing a try, except in code than it is in tests